### PR TITLE
Default document azure storage

### PR DESCRIPTION
**Pull request by jrudley**
In a comment on the pull request, include the following information and check off the list.

 - [ ] Mini bio
  - Soft capped to 75 words
 - [ ] Desired date of publishing 06-01-2018 (asap)
  - Make sure this matches the `date` in the front matter
 - [ ] Social media contact info twitter: jrudley
 - [ ] Redaction: (yes|no) yes
  - Allow redaction, editing or structure changes by technical writer staff
 - [ ] Comments: (yes|no) no
  - If you're willing to answer comments about your post, set `comments: true`.
  - Make sure to subscribe to your post comments by clicking on the **Subscribe** link at the very bottom of the post _once_ it's published.

@catlook Please review.

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-dd1c1bc118/)

**Comment by jrudley** [comment]
hrm, it seems the ``` ticks did not like the html code formatting

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-dd1c1bc118/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-23ca0677c7/)

**Comment by jrudley** [comment]
ok, fixed the formatting :)

**Comment by catlook** [comment]
reviewing!!

File: _posts/2018-06-01-Default-Document-Azure-Storage-Account.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+date: 2018-06-01 00:00
+comments: true
+author: Jimmy Rudley
+published: true
+authorIsRacker: true
+categories:
+    - General
+---
+
+Azure has provided the functionality to host static websites from an Azure Storage Account for some time, but it did not support setting a default document. This functionality is in preview and should hit public preview this month. Let's take a look on how to test out this great feature.
```
Comment (catlook) [inline _posts/2018-06-01-Default-Document-Azure-Storage-Account.md]: Azure provided the functionality to host static websites from an Azure Storage Account, but ...

File: _posts/2018-06-01-Default-Document-Azure-Storage-Account.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+<!-- more -->
+
+Microsoft has not announced the public preview as of today, but to access it, please click this [link](http://aka.ms/staticwebsites) to enable static website support in the portal. Create a new storage account and make sure to keep the account kind **StorageV2** and the location **West Central US**. ![storage account]({% asset_path 2018-06-01-Default-Document-Azure-Storage-Account/create.png %}) Once the Storage Account resource has been created, open up the storage account from the blade. A Static website (preview) setting will now display. Click on it then select **Enabled** which will bring up 2 text boxes to set your **Index document name** and **Error document path**. The **Index document name** is the default document that will be selected when a user browsers to the primary endpoint generated. The **Error document path** will be the 404 page. Select **Save** after filling in the file names you will be using. ![configure]({% asset_path 2018-06-01-Default-Document-Azure-Storage-Account/configure.png %})
```
Comment (catlook) [inline _posts/2018-06-01-Default-Document-Azure-Storage-Account.md]: ... public preview by this blog's publish date, but, please click ...
[Ed.:  I suggest reformatting the steps like the following to make them easier to understand]

Use the following steps to set up a static wesite:

1. Create a new storage account and make sure to set the account kind to ``StorageV2`` and the location to ``West Central US``. storage account 
2. After the Storage Account resource has been created, open up the storage account from the blade, which displays a Static website (preview) setting window. 
3. Click on the window and select **Enabled**, which brings up 2 text boxes to set the ``Index document name`` and the ``Error document path``. The ``Index document name`` is the default document that is selected when a user browses to the primary endpoint that was generated. The ``Error document path`` is the 404 page. 
4. Fill in the file names you plan to use and select **Save**. configure

File: _posts/2018-06-01-Default-Document-Azure-Storage-Account.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+I created a sample index.html to generate the date.
```
Comment (catlook) [inline _posts/2018-06-01-Default-Document-Azure-Storage-Account.md]: I created the following sample index.html ...

File: _posts/2018-06-01-Default-Document-Azure-Storage-Account.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    <title>uh oh! </title>
+</head>
+<body>
+     Looks like you hit a 404, please try again:
+</body>
+</html>
+
+```
+
+I noticed that I cannot find the web container when using Azure Storage Explorer. The workaround is to click the **$web** container and use the browser preview functionality to upload the index document name and error document files to the **$web** container.
```
Comment (catlook) [inline _posts/2018-06-01-Default-Document-Azure-Storage-Account.md]: I noticed that I couldn't find the web container when using Azure Storage Explorer, but I could click the **$web** container and use the browser preview functionality to upload the index document name and error document files to the **$web** container.

File: _posts/2018-06-01-Default-Document-Azure-Storage-Account.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+Browsing to the primary endpoint URL, I now have a default document being used. ![default]({% asset_path 2018-06-01-Default-Document-Azure-Storage-Account/default.png %}) I also tested out a 404 error and that works as well. ![404]({% asset_path 2018-06-01-Default-Document-Azure-Storage-Account/404.png %}) You can download the html files from this [repo]( https://github.com/jrudley/staticwebsite)
```
Comment (catlook) [inline _posts/2018-06-01-Default-Document-Azure-Storage-Account.md]: When I browse to the primary endpoint URL, I see my default document. ...
I also tested out a 404 error, which worked as expected. ...
[Ed: put a period after your repo link :)]

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-a3cee362de/).

**Comment by jrudley** [comment]
Thanks @catlook I made the changes as suggested :)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-a3cee362de/)

**Pull request merged**